From e2d92b5adf710bacaea14e3fbca936997c6b9469 Mon Sep 17 00:00:00 2001
From: Harald Hoyer <harald@redhat.com>
Date: Thu, 20 Oct 2011 12:13:50 +0200
Subject: [PATCH] */module-setup.sh: turn off debugging for module search

turn off debugging for module search, because it clutters the debug log
---
 modules.d/40network/module-setup.sh        |    2 ++
 modules.d/90kernel-modules/module-setup.sh |    2 ++
 modules.d/90multipath/module-setup.sh      |    2 ++
 3 files changed, 6 insertions(+), 0 deletions(-)

diff --git a/modules.d/40network/module-setup.sh b/modules.d/40network/module-setup.sh
index eb7ef9b..fbf6f34 100755
--- a/modules.d/40network/module-setup.sh
+++ b/modules.d/40network/module-setup.sh
@@ -52,7 +52,9 @@ installkernel() {
             done | nmf1 1>&${_merge}
         }
         # Use two parallel streams to filter alternating modules.
+        set +x
         eval "( ( rotor ) ${_side2}>&1 | nmf1 ) ${_merge}>&1"
+        [[ $debug ]] && set -x
     }
 
     find_kernel_modules_by_path drivers/net | net_module_filter | instmods
diff --git a/modules.d/90kernel-modules/module-setup.sh b/modules.d/90kernel-modules/module-setup.sh
index 6e3a918..db4b276 100755
--- a/modules.d/90kernel-modules/module-setup.sh
+++ b/modules.d/90kernel-modules/module-setup.sh
@@ -31,7 +31,9 @@ installkernel() {
                 done | bmf1 1>&${_merge}
             }
             # Use two parallel streams to filter alternating modules.
+            set +x
             eval "( ( rotor ) ${_side2}>&1 | bmf1 ) ${_merge}>&1"
+            [[ $debug ]] && set -x
         }
         hostonly='' instmods sr_mod sd_mod scsi_dh scsi_dh_rdac scsi_dh_emc
         hostonly='' instmods pcmcia firewire-ohci
diff --git a/modules.d/90multipath/module-setup.sh b/modules.d/90multipath/module-setup.sh
index f68b58d..43f0a3d 100755
--- a/modules.d/90multipath/module-setup.sh
+++ b/modules.d/90multipath/module-setup.sh
@@ -33,6 +33,7 @@ depends() {
 }
 
 installkernel() {
+    set +x
     mp_mod_filter() {
         local _mpfuncs='scsi_register_device_handler|dm_dirty_log_type_register|dm_register_path_selector|dm_register_target'
         local _f
@@ -45,6 +46,7 @@ installkernel() {
 
     ( find_kernel_modules_by_path drivers/scsi;
       find_kernel_modules_by_path drivers/md )  |  mp_mod_filter  |  instmods
+    [[ $debug ]] && set -x
 }
 
 install() {
